src/main/java/core/basesyntax/service/impl/StorageServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.StorageService;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
+    private final Storage storage = new Storage();
+
+    public StorageServiceImpl() {
+    }
+
+    @Override
+    public void save(String name, Integer quantity) {
+        storage.getProducts().put(name, quantity);
+    }
+
+    @Override
+    public Integer get(String key) {
+        return storage.getProducts().get(key);
+    }
+
+    @Override
+    public Map<String, Integer> getAll() {
+        return storage.getProducts();
+    }
+
+    @Override
+    public void update(String name, Integer value) {
+        storage.getProducts().put(name, value);
+    }
```

Comment:
save and update are identical, remove one of them

Reply:
i agree. I've deleted update method